.github/workflows/python-ci.yml

```
@@ -21,39 +21,59 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        runs-on: [ubuntu-latest]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        flint-inst-cmd: ["sudo apt-get install libflint-dev"]
         include:
-          - runs-on: ubuntu-latest
-            flint-inst-cmd: sudo apt-get install libflint-dev
           - runs-on: macos-latest
             flint-inst-cmd: brew install flint
+            python-version: 3.8
+          - runs-on: macos-latest
+            flint-inst-cmd: brew install flint
+            python-version: 3.11
```

Not 100% sure, but I think this could be simplified to
```suggestion
            python-version: ["3.8", "3.11"]
```